airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresQueryUtils.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.postgres;
+
+import java.sql.SQLException;
+
+/**
+ * Utility class to define constants related to querying postgres
+ */
+public class PostgresQueryUtils {
+
+  public static final String NULL_CURSOR_VALUE_WITH_SCHEMA =
```

Comment:
Minor nit: since both `NULL_CURSOR_VALUE_WITH_SCHEMA` and `NULL_CURSOR_VALUE_NO_SCHEMA` are table queries, can we add `_QUERY` to the end to keep it in alignment with `TABLE_ESTIMATE_QUERY`? 

It also helps to quickly know this is a database query string

Reply:
Done